docs/guides/configuration/index.md

```diff
@@ -65,19 +66,46 @@ marimo searches for the `.marimo.toml` file in the following order:
 If no `.marimo.toml` file is found, marimo creates one for you in an XDG config
 compliant way.
 
-View your current configuration and locate the config file with:
+To view your current configuration and locate the config file, run:
 
 ```bash
 marimo config show
 ```
 
+To describe the user configuration options, run:
+
+```bash
+marimo config describe
+```
+
+### Overriding settings with pyproject.toml
+
+You can override user configuration settings with a `pyproject.toml` file. This
+is useful for sharing configurations across teams or ensuring consistency across
+notebooks. You must edit the `pyproject.toml` file directly to override settings.
+
+For example, the following `pyproject.toml` file overrides the `autosave` setting
+in the user configuration:
+
+```toml
+[tool.marimo.save]
+autosave = "after_delay"
```

Comment:
Small thing. Should this example be tailored to things teams might want to standardize? Such as

```toml
[tool.marimo.format]
line_length = 120

[tool.marimo.display]
default_width = "full"
```